### PR TITLE
Add tool to help construct the JDBC URL.

### PR DESCRIPTION
**Pull request by laixer**

### CODE REVIEW

File: examples/getting-started/src/main/java/com/google/cloud/sql/mysql/example/GettingStarted.java [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.sqladmin.SQLAdmin;
+import com.google.api.services.sqladmin.model.DatabaseInstance;
+import com.google.api.services.sqladmin.model.InstancesListResponse;
+import com.google.common.collect.ImmutableSet;
+
+import java.io.Console;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.sql.*;
```
Comment (hfwang) [inline examples/getting-started/src/main/java/com/google/cloud/sql/mysql/example/GettingStarted.java]: I'd prefer you import classes explicitly.

File: examples/getting-started/src/main/java/com/google/cloud/sql/mysql/example/GettingStarted.java [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
```
Comment (hfwang) [inline examples/getting-started/src/main/java/com/google/cloud/sql/mysql/example/GettingStarted.java]: This requires Java 8, will that be a problem?

File: examples/getting-started/src/main/java/com/google/cloud/sql/mysql/example/GettingStarted.java [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+    for (int i = 0; i < instances.size(); i++) {
+      DatabaseInstance databaseInstance = instances.get(i);
+      System.out.println(String.format("%d: %s (%s)", i + 1, databaseInstance.getName(),
+          databaseInstance.getConnectionName()));
+    }
+
+    String instanceConnectionName = "";
+    while (instanceConnectionName.isEmpty()) {
+      System.out.print("Please enter the number of the instance you want to use [1]: ");
+      String line = console.readLine();
```
Comment (hfwang) [inline examples/getting-started/src/main/java/com/google/cloud/sql/mysql/example/GettingStarted.java]: Might be worth extracting the "select from a list" logic into its own method. It'd only have two users though.

File: examples/getting-started/src/main/java/com/google/cloud/sql/mysql/example/GettingStarted.java (hunk 1 above)
Comment (laixer) [inline examples/getting-started/src/main/java/com/google/cloud/sql/mysql/example/GettingStarted.java]: Done, wasn't intentional. #justintellijthings

File: examples/getting-started/src/main/java/com/google/cloud/sql/mysql/example/GettingStarted.java (hunk 2 above)
Comment (laixer) [inline examples/getting-started/src/main/java/com/google/cloud/sql/mysql/example/GettingStarted.java]: Should be fine for aux tools.

**Pull request merged**